tests/driver_lis2dh12/main.c

```diff
@@ -48,6 +79,25 @@ int main(void)
         return 1;
     }
 
+    /* enable interrupt Pins */
+#ifdef LIS2DH12_INT_PIN1
+    if (gpio_init_int(LIS2DH12_INT_PIN1,GPIO_IN, GPIO_RISING,lis2dh12_int_cb,"INT_1") == -1)
+        puts("init_int failed!\n");
+#endif
+#ifdef LIS2DH12_INT_PIN2
+    if (gpio_init_int(LIS2DH12_INT_PIN2,GPIO_IN, GPIO_RISING,lis2dh12_int_cb,"INT_2") == -1)
+        puts("init_int failed!\n");
+#endif
+
+    /* create and set the interrupt params */
+    lis2dh12_int_params_t params = {0};
+    params.int_type = 0b01000000;
+    params.int_config = 1;
+    params.int_threshold = 0b00011111;
```

Comment:
This section should be guarded and executed only if interrupt lines are defined.